### DISCUSSION TITLE
Microsoft Defender detecting as hacktool

### DISCUSSION THREAD

**Discussion by Javican** [valinet/ExplorerPatcher#2904] [General]
just providing information that Microsoft defender (office 365) is detecting this as a hacktool

![image](https://github.com/valinet/ExplorerPatcher/assets/7501860/9b62bd2b-18ac-4e0e-8acf-808f100f90cd)

**Comment by Amrsatrio** [comment]
Please install the latest [63.4 prerelease](https://github.com/valinet/ExplorerPatcher/releases/tag/22621.3007.63.4_91aaa82) or [62.2 release from two months ago](https://github.com/valinet/ExplorerPatcher/releases/tag/22621.2861.62.2_9b68cc0). Both are not flagged by MS. Do not continue to use 63.2 and 63.3 as they have been falsely flagged as malicious by MS.
